HopsanGUI/PlotTab.cpp

```diff
@@ -1397,28 +1397,29 @@ void PlotTab::openFrequencyAnalysisDialog(PlotCurve *pCurve)
     double minX = pCurve->getSharedTimeOrFrequencyVariable()->minOfData();
     double maxX = pCurve->getSharedTimeOrFrequencyVariable()->maxOfData();
 
-    if(minX == maxX) {
+    QLabel *pMinTimeLabel = new QLabel("Min time: ", pDialog);
+
+    double xdiff = maxX-minX;
+    if(0 == xdiff) {
```

Comment:
Hmmm, is this not the same thing as equals? maybe not exactly due to subtraction? but maybe equals comparison is the right thing in this case anyway.

Reply:
Comparing double to zero should always be safe as far as I know. It will at least prevent division by zero, which is the purpose here.